src/mame/drivers/missb2.cpp

```diff
@@ -440,27 +440,24 @@ MACHINE_START_MEMBER(missb2_state,missb2)
 	m_gfxdecode->gfx(1)->set_palette(*m_bgpalette);
 
 	save_item(NAME(m_sound_nmi_enable));
-	save_item(NAME(m_pending_nmi));
-	save_item(NAME(m_sound_status));
 	save_item(NAME(m_video_enable));
 }
 
 MACHINE_RESET_MEMBER(missb2_state,missb2)
 {
+	MACHINE_RESET_CALL_MEMBER(common);
```

Comment:
Could this be done better by overriding `driver_reset()` and calling base class methods where appropriate?  It's generally better to split state classes up than to add messy speicalised `MACHINE_RESET_MEMBER` etc. that call each other.